resources/lang_chs.yml

```diff
@@ -0,0 +1,88 @@
+# Chinese Simplifyed Translation
+# Not Fully Checked
+# Authors:
+# Thunder33345
+
+#Join
+join-message: "&6此服务器需要密码认真."
+new-player: "&e欢迎 {player} 来到此服务器"
+
+#Login
+login-message: "&6请登入通过 /login <密码>\n&忘记您的密码? 用 /forgotpassword <pin> <新密码>"
+login-popup: "&6请登入."
+login-tip: "&6请登入."
+login-boss-bar: "&6请登入."
+authentication-success: "&a登入成功."
+authentication-success-uuid: "&a自动登入(UUID)." #need help for better translation
+authentication-success-xbox: "&a自动登入(Xbox Live)." #need help for better translation
+attempted-logins: "&6您账户已有 {attempts} 登入尝试." #need help for better translation
+already-authenticated: "&c您已经登入."
+incorrect-password: "&c密码不对. 剩下 {tries} 尝试."
+not-registered: "&c您还没注册."
+too-many-tries: "&c太多登入尝试"
+timeout-message: "&c登入超时"
+too-many-on-ip: "&c太多用户有同样IP地址."
+
+#Register
+register-message: "&6请注册通过 /register <密码> <确认密码> [电邮]"
+register-popup: "&6请注册."
+register-tip: "&6请注册."
```

Comment:
You are missing register-title, register-subtitle, & register-actionbar (Same as register-popup & register-tip).